### PR TITLE
Task #48976 [Admin] Create Room

### PR DESCRIPTION
**Pull request by anhlh-3049**
## Related Tickets
- [#TicketID](https://edu-redmine.sun-asterisk.vn/issues/???)

## WHAT (optional)
- Create Room

## Evidence (Screenshot or Video)
Screenshot_7

### CODE REVIEW

**Comment by anhlh-3049** [comment]
ready

File: app/views/admin/rooms/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<% provide :title, t(".provide_title") %>
+<h2 class="col-md-6 offset-3"><%= t ".title" %></h2>
+<div class="col-md-6 offset-3">
+  <%= form_for([:admin, @room]) do |f| %>
+
+    <%= f.label t(".name"), class: "text-color" %>
+    <%= f.text_field :name, class: "form-control mt-2" %>
+
+    <%= f.label t(".price") %>
+    <%= f.number_field :price, class: "form-control" %>
```
Comment (ngocvt-0484) [inline app/views/admin/rooms/new.html.erb]: set min

File: app/views/admin/rooms/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<% provide :title, t(".provide_title") %>
+<h2 class="col-md-6 offset-3"><%= t ".title" %></h2>
+<div class="col-md-6 offset-3">
+  <%= form_for([:admin, @room]) do |f| %>
```
Comment (ngocvt-0484) [inline app/views/admin/rooms/new.html.erb]:
```suggestion
  <%= form_for [:admin, @room] do |f| %>
```

File: app/views/admin/rooms/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+    <%= f.label t(".name"), class: "text-color" %>
+    <%= f.text_field :name, class: "form-control mt-2" %>
+
+    <%= f.label t(".price") %>
+    <%= f.number_field :price, class: "form-control" %>
+
+    <%= f.label t(".description") %>
+    <%= f.text_area :description , class: "form-control" %>
```
Comment (ngocvt-0484) [inline app/views/admin/rooms/new.html.erb]: set max

Comment (ngocvt-0484) [reply app/views/admin/rooms/new.html.erb]: thêm validate

File: app/controllers/admin/rooms_controller.rb [hunk 4]
```diff
@@ -1,8 +1,21 @@
-class Admin::RoomsController < Admin::AdminController
```
Comment (ngocvt-0484) [inline app/controllers/admin/rooms_controller.rb:1]: sao lại xóa

**Comment by anhlh-3049** [comment]
ready

File: app/views/admin/rooms/new.html.erb (hunk 3 above)
Comment (ngocvt-0484) [reply app/views/admin/rooms/new.html.erb]: chưa fix

**Comment by anhlh-3049** [comment]
ready

File: app/views/admin/rooms/new.html.erb [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+      class: "form-control", max: Settings.admin.new.description.max_length %>
+  </div>
+  <br>
+  <div class="field">
+    <%= f.label :room_view_id %>
+    <%= f.select :room_view_id, RoomView.pluck(:name, :id) , class: "form-control" %>
+  </div>
+  <div id='images'>
+    <%= f.fields_for :images do |image| %>
+        <%= render "image_fields", f: image %>
```
Comment (ngocvt-0484) [inline app/views/admin/rooms/new.html.erb:26]: indent

File: app/views/admin/rooms/new.html.erb [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+    <br>
+    <% end %>
+    <div class="links">
+      <%= link_to_add_association t(".add_image"), f, :images %>
+    </div>
+  </div>
+    <hr>
+    <%= f.submit t(".btn_create"), class: "btn btn-primary" %>
```
Comment (ngocvt-0484) [inline app/views/admin/rooms/new.html.erb:34]: indent

File: app/views/admin/rooms/new.html.erb (hunk 5 above)
Comment (ngocvt-0484) [reply app/views/admin/rooms/new.html.erb:26]: xem lại indent

File: app/views/admin/rooms/new.html.erb [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+    <%= f.label t(".description") %>
+    <%= f.text_area :description,
+      class: "form-control", max: Settings.admin.new.description.max_length %>
+  </div>
+  <br>
+  <div class="field">
+    <%= f.label :room_view_id %>
+    <%= f.select :room_view_id, RoomView.pluck(:name, :id) , class: "form-control" %>
+  </div>
+  <div id='images'>
```
Comment (ngocvt-0484) [inline app/views/admin/rooms/new.html.erb]: nháy "

**Pull request merged**